### DISCUSSION TITLE
Very nice product, congrats to the founder!

### DISCUSSION THREAD

**Discussion by vitalijalbu** [cschroeter/park-ui#36] [General]
Very nice product, congrats to the founder! Do you plan to extend components in future? Creating maybe a showcase page and adding more templates?
Nice job!!!

**Comment by coopbri** [comment]
Not sure if you've seen this already, but Christian has a great showcase of examples/templates on the home page: https://park-ui.com/ (scroll to bottom)

As for component extension and adding more templates, there is no doubt more will be added but this is a relatively new offering (as are Zag, Ark, and Panda). With that being said, all packages (Park UI & the aforementioned) are being actively developed.
